### PR TITLE
SC-8454 - delay search

### PR DESCRIPTION
**Pull request by virgilchiriac**
# Description

Adds a delay in the Lern-Store search.

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-8454

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: src/pages/content/index.vue [hunk 1]
```diff
@@ -196,8 +196,11 @@ export default {
 			}
 		},
 		enterKeyHandler() {
-			this.searchContent();
-			this.activateTransition = true;
+			clearTimeout(this.$options.debounce2);
+			this.$options.debounce2 = setTimeout(() => {
+				this.searchContent();
+				this.activateTransition = true;
+			}, 500);
```
Comment (JohnKoumarelas) [inline src/pages/content/index.vue:203]: From 0ms to 500ms waiting time between keystrokes seems a more reasonable configuration.

**Review by JohnKoumarelas — APPROVED** [review]
From 0ms to 500ms waiting time between keystrokes seems a more reasonable configuration.

File: src/pages/content/index.vue (hunk 1 above)
Comment (Ojself) [reply src/pages/content/index.vue:203]: What happens on the first key event? Is it clearing a non existing timeout?

Comment (virgilchiriac) [reply src/pages/content/index.vue:205]: fixed

**Review by Ojself — APPROVED** [review]

**Review by Ojself — APPROVED** [review]

**Review by JenyMzo — APPROVED** [review]

**Pull request merged**